tests/unit/validation/test_utils.py

```diff
@@ -6,7 +6,6 @@
 import pytest
 
 from power_grid_model import initialize_array
-from power_grid_model.data_types import BatchDataset, BatchList
```

Comment:
Does pylint not catch this :/

Reply:
nope because the tests are not `pylint`ed